example/spec/parallel/helloWorld/HelloWorldECSServiceAutoScalingSpec.js

```diff
@@ -0,0 +1,41 @@
+const range = require('lodash.range');
+const { sleep } = require('@cumulus/common/util');
+const { buildAndExecuteWorkflow } = require('@cumulus/integration-tests');
+const { loadConfig } = require('../../helpers/testUtils');
+const { getEcsClusterService, getServiceEvents } = require('../../helpers/ecsUtils');
+
+const awsConfig = loadConfig();
+
+describe('The Hello World workflow using ECS and CMA Layers', () => {
+  let workflowExecutions = [];
+  const startTime = new Date();
+
+  beforeAll(async () => {
+    const workflows = range(15).map(() =>
+      buildAndExecuteWorkflow(
+        awsConfig.stackName,
+        awsConfig.bucket,
+        'EcsHelloWorldWorkflow',
+        null,
+        null,
+        { taskDurationInSecs: 15 }
+      ));
+    workflowExecutions = await Promise.all(workflows);
+  });
+
+  it('executes successfully', () => {
+    workflowExecutions.forEach((workflowExecution) => {
+      expect(workflowExecution.status).toEqual('SUCCEEDED');
+    });
+  });
+
+  it('performs ECS Service Autoscaling', async () => {
+    // wait for the event messages
+    await sleep(180 * 1000);
```

Comment:
Where did we get this sleep time from? Is it possible to introduce an intermittent issue here where the services spin up then spin down within that amount of time? or take longer to spin up?

Reply:
The numbers are based on my testing.  When the workflow executions are queued up, the ECS service should spin up, and after the executions are complete, the service should spin down.  The 180 seconds is for waiting the service to spin down and event messages are available.  We could wait longer for the service to spin down.
I tried to check the `desiredCount/desiredCount` of the service, but that requires to call `describeServices` frequently, and the counts are changing when the service spins up and down.